### PR TITLE
Create example.ipynb

### PR DESCRIPTION
**Pull request by AlysenTownsley**
Updated example (vignette) with content from the google doc and ran notebook to confirm functionality / outputs.

### CODE REVIEW

File: docs/example.ipynb [hunk 1]
```diff
@@ -2,23 +2,429 @@
...
+            ],
+            "source": [
+                "interest_calc(5000, 1, 5.5)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "This returns the values 5.5 and 275.00. This refers to a GIC rate of 5.5% and an interest accrual of $275.00, respectively. Hmm…this is $30 more than Bank A. Eva knew that her shrewdness would pay off! 🕵️‍♀️ \n",
```
Comment (tonyshumlh) [inline docs/example.ipynb:111]: You may want to escape Latex here with 
```suggestion
 ... \\$275.00, respectively. Hmm…this is \\$30 ...
```

File: docs/example.ipynb [hunk 2]
```diff
@@ -2,23 +2,429 @@
...
+                "interest_calc(5000, 3)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "This returns the values 4.0 and 624.32. This refers to a GIC rate of 4.0% and an interest accrual of $624.32, respectively. Wowza, what a difference! 🤑\n",
+                "\n",
+                "Eva is shrewd with her money, though, and wants to compare the rates at a different bank just to check if she can get a better deal (let’s see if she can squeeze a few more bucks out of this investment)! In order to do this, she’ll need to enter Bank B’s interest rates manually in the interest_calc function. Bank B is offering 5.5% GIC rates for a 1 year fixed term, and 4.00% for a 3 year term. She completes this in the below step:"
```
Comment (tonyshumlh) [inline docs/example.ipynb:84]: It seems to be 3.5% for 3 year term
```suggestion
... 3.5% for a 3 year term. ..."
```

File: docs/example.ipynb [hunk 3]
```diff
@@ -2,23 +2,429 @@
...
+            ],
+            "source": [
+                "interest_calc(5000, 3, 3.5)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "This returns the values 3.5 and 543.59. This refers to a GIC rate of 3.5% and an interest accrual of $543.59, respectively. Hmm…this is about $80.73 less than Bank A’s return after 3 years. Oh well, it’s always good to double check. 🤷‍♀️ At least Eva knows that she’s getting the best return with Bank A! 💸"
```
Comment (tonyshumlh) [inline docs/example.ipynb:140]:
```suggestion
... \\$543.59, respectively. Hmm…this is about \\$80.73 ...
```

File: docs/example.ipynb [hunk 4]
```diff
@@ -2,23 +2,429 @@
...
+            "source": [
+                "The chart shows that the interest from Bank A's GIC is actually higher than the friend's suggested investment. Seemingly, the “best thing” might not always be good! 🤔 Eva decides to go ahead and purchase the 3-year GIC from Bank A. Great job on the analysis Eva!!"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "#### Aside\n",
+                "Now for the most important math (outside of the scope of the `gic_calculator` package, but of dire personal importance to Eva) - at an average pizza price of $20.00 for a large pizza, will Eva be able to survive the semester? Assume that 1 large pizza will last Eva 3 meals. At an interest accrual of $624.32, Eva will be able to purchase 31 pizzas, which will last her a total of 93 meals. Assuming that Eva eats 3 nutritious pizza meals per day, she will only have enough money for 31 days of pizza! Oh no!! Eva sighs 😞 … I guess that she’ll have to get a job at Domino’s (the employee discount will come in very handy!). \n",
```
Comment (tonyshumlh) [inline docs/example.ipynb:424]:
```suggestion
 ... \\$20.00 for a large pizza, will Eva be able to survive the semester? Assume that 1 large pizza will last Eva 3 meals. At an interest accrual of \\$624.32 ...
```

**Comment by tonyshumlh** [comment]
Tested. Comments are written

**Comment by AlysenTownsley** [comment]
Reviewed Tony's comments - agreed and committed all. Thanks for the thorough review @tonyshumlh!

**Review by sungg888 — APPROVED** [review]
Reviewed and look good to me.

**Pull request merged**